### ISSUE TITLE
Invalid object referenced when reconnecting while on splash screen

### ISSUE THREAD

**Issue by MysteryPoo** [MysteryPoo/DD-Arena#1]
___________________________________________
############################################################################################
FATAL ERROR in
action number 1
of Async Event: Networking
for object oLobbyClient:

Pop :: Execution Error - Variable references invalid object (-4).<unknown variable>
 at gml_Script_Lobby_HandleHandshake (line 45) -               mButtonOnline.mIsEnabled = true;
############################################################################################
--------------------------------------------------------------------------------------------
stack frame is
gml_Script_Lobby_HandleHandshake (line 45)
called from - gml_Script_Lobby_ProcessPacket (line 12) -               Lobby_HandleHandshake(argument2);
called from - gml_Object_oLobbyClient_Other_68 (line 34) -               Lobby_ProcessPacket(_command, _commandSize, _data);

Steps to Reproduce:
* Ensure the DDAMMS is offline
* Start the GameClient (You can expect to get d/c warnings)
* Bring the DDAMMS online
* When the GameClient connects, this error is hit.

**Comment by jnrbrain** [comment]
Did u tried use “if exist”

**Comment by MysteryPoo** [comment]
Yeah, that’s a workaround however I want to figure out why it’s getting set the initial value of “noone”. It’s probably an obvious thing, but I just simply haven’t looked at it at all and left this here as a reminder.

**Comment by jnrbrain** [comment]
I got same error in my project, but i solved this with global variables and local variables. Maybe you must check it :?
